client/events_test.go

```diff
@@ -158,6 +166,9 @@ func TestEvents(t *testing.T) {
 
 				break loop
 			case e := <-messages:
+				if e.ID == "" {
```

Comment:
Wouldn't this mean the test could complete without receiving a single of the expected events?

Reply:
You are right, I'll modify it, thx.